### PR TITLE
Deprecate old code and update README

### PR DESCRIPTION
**Pull request by bjfletcher**
new, replacement code lives in https://github.com/Financial-Times/next-article-email-api

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,86 +1,5 @@
-# Next Article Send By Email
+# next-rtcl-email-api
 
-[![Circle CI](https://circleci.com/gh/Financial-Times/next-rtcl-email-api/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/next-rtcl-email-api)
+Deprecated - now lives in https://github.com/Financial-Times/next-article-email-api
```
Comment (andygout) [inline README.md]: Is there any requirement having a dead repo lying round pointing to another one? Is there anything to stop us from deleting it altogether (first time I've seen a repo sunsetted)?

**Comment by andygout** [comment]
👍  Suppose this is one of the very few instances that we don't need to worry about failing tests 😉

File: README.md (hunk 1 above)
Comment (bjfletcher) [inline README.md]: https://github.com/Financial-Times/n-third-party-code

I quite like that approach. We keep the old code in the history, with commits and all. But in the `master` branch, it's made quite clear that it's deprecated and where its replacement is.

Comment (andygout) [inline README.md]: 👍

**Pull request merged**